assets/src/edit-story/components/library/panes/media/media3p/media3pPane.js

```diff
@@ -53,12 +53,50 @@ const ProviderTabSection = styled.div`
   padding: 0 24px;
 `;
 
+// TODO:(https://github.com/google/web-stories-wp/issues/2360)
+// Category should be collapsible and expandable.
 const CategorySection = styled.div`
   background-color: ${({ theme }) => theme.colors.bg.v3};
   min-height: 94px;
   padding: 30px 24px;
+  display: flex;
+  flex-wrap: wrap;
 `;
 
+export const Pill = styled.span`
```

Comment:
The Media3pPane is getting quite busy, and there will be a lot of logic required to render the categories and respond to events.

Can all of the category things be moved to a separate file?

Reply:
Done. I moved the Pill things into a seperate file as well.